pkg/agent/addon_status_controller.go

```diff
@@ -0,0 +1,183 @@
+package agent
```

Comment:
Missing Copyright.

Reply:
Josh, the other files in the folder didn't have the license header that's why I didn't include it in the new ones.
https://github.com/stolostron/hypershift-addon-operator/blob/main/pkg/agent/agent.go
https://github.com/stolostron/hypershift-addon-operator/blob/main/pkg/agent/clusterclaim.go

I guess we should go over all the files and add the license header or just leave it?